xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -92,11 +92,17 @@ void CGUIWindowPVRGuide::GetContextButtons(int itemNumber, CContextButtons &butt
     CPVRTimerInfoTagPtr timer(epg->Timer());
     if (timer)
     {
+      if (timer->GetTimerScheduleId() != PVR_TIMER_NO_PARENT)
+        buttons.Add(CONTEXT_BUTTON_EDIT_TIMER_RULE, 19243); /* Edit timer rule */
```

Comment:
Yes, for the UI it is. Discussed this with @da-anda in the PR that recently introduced separate windows for oneshot timers ("Timers") and repeating timers ("Timer rules"). 

Internally, we used so far 'repeating timer'. Do you want me to change the internal naming from timer rule to repeating timer?